code/modules/admin/verbs/vox_raiders.dm

```diff
@@ -1,65 +0,0 @@
-var/global/vox_tick = 1
-
-/mob/living/carbon/human/proc/equip_vox_raider()
-
-	var/obj/item/device/radio/R = new /obj/item/device/radio/headset/syndicate(src)
-	R.set_frequency(SYND_FREQ) //Same frequency as the syndicate team in Nuke mode.
-	equip_to_slot_or_del(R, slot_l_ear)
-
-	equip_to_slot_or_del(new /obj/item/clothing/under/vox/vox_robes(src), slot_w_uniform)
```

Comment:
I would really rather you don't just kill off this proc entirely and that you add all of these loadouts as actual admin loadouts, these aren't useless